isis/src/base/apps/framestitch/framestitch.xml

```diff
@@ -127,6 +132,30 @@
         <internalDefault>Camera</internalDefault>
         <minimum>1</minimum>
       </parameter>
+
+      <parameter name="NUM_LINES_OVERLAP">
+        <type>integer</type>
+        <brief>
+          The number of lines in each frame repeating information
+          present in neighboring frames.
+        </brief>
+        <description>
+          <p>
+          The estimated total number of lines in each frame which have
+          the same pixel values as frames above and below it. During
+          processing, half this amount of lines will be removed from
+          each of the top and bottom of the frame. Must be an even
+          non-negative number.
+          </p>
+          <p>
+            If not entered, the program will assume the value 0. A value
+			of 2 or 4 works reasonably for WAC images.
+          </p>
+        </description>
+        <internalDefault>Camera</internalDefault>
```

Comment:
Change this to an actual default of 0

Reply:
I made the default to be 0 and tested that the tool can handle being run with this value not explicitly set when it then uses the default.